parts/plugins/x-php.py

```diff
@@ -59,6 +59,7 @@ def schema(cls):
         return schema
 
     def __init__(self, name, options, project):
+        options.configflags.extend(['--with-libdir=lib/{}'.format(project.arch_triplet)])
```

Comment:
I don't think this will be correct on 64-bit systems. From [here](https://secure.php.net/manual/en/configure.about.php):

> Specifies the directory where the libraries to build PHP exists on a Unix system. For 64bit systems, its needed to specify this argument to the lib64 directory like: --with-libdir=lib64.